fixtures/mega-lite/framebot-600.json

```diff
@@ -0,0 +1,854 @@
+          "dmxRange": [0, 19],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [20, 29],
+          "type": "Maintenance",
+          "comment": "Reset All",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [30, 39],
+          "type": "Maintenance",
+          "comment": "Reset Pan/Tilt",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [40, 49],
+          "type": "Maintenance",
+          "comment": "Reset Color/Gobo/CMYK",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [50, 59],
+          "type": "Maintenance",
+          "comment": "Reset Zoom/Focus/Prism/Frost",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [60, 69],
+          "type": "Maintenance",
+          "comment": "Reset Colormation/Iris/Framing",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [70, 79],
+          "type": "Maintenance",
+          "comment": "Display Default",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [80, 89],
+          "type": "Maintenance",
+          "comment": "Display Off",
+          "hold": "8s"
+        },
+        {
+          "dmxRange": [90, 255],
+          "type": "NoFunction"
+        }
+      ]
+    },
+    "Frame 1A": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "Frame 1A Engage"
+      }
+    },
+    "Frame 1B": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "Frame 1B Engage"
+      }
+    },
+    "Frame 2A": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "Frame 2A Engage"
+      }
+    },
+    "Frame 2B": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "Frame 2B Engage"
+      }
+    },
+    "Frame 3A": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "Frame 3A Engage"
+      }
+    },
+    "Frame 3B": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "Frame 3B Engage"
+      }
+    },
+    "Frame 4A": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "Frame 4A Engage"
+      }
+    },
+    "Frame 4B": {
+      "defaultValue": 0,
+      "capability": {
+        "type": "Generic",
+        "comment": "Frame 4B Engage"
+      }
+    },
+    "Frame Rotation": {
+      "fineChannelAliases": ["Frame Rotation fine"],
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 2039],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [2040, 61200],
+          "type": "Generic",
+          "comment": "Frame Rotation"
+        },
+        {
+          "dmxRange": [61201, 65535],
+          "type": "Generic",
+          "comment": "Frame Bounce"
+        }
+      ]
+    },
```

Comment:
Please use [`BladeSystemRotation`](https://github.com/OpenLightingProject/open-fixture-library/blob/master/docs/capability-types.md#blade-system-rotation) for these capabilities.